AccelGyro accelGyro
float64[3] magnetometer_value
